tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -182,6 +184,23 @@ function test_wp_enqueue_script_filter_null_version() {
 		), $this->classicpress_asset_version_calls );
 	}
 
+	/**
+	 * @see https://core.trac.wordpress.org/ticket/42804
+	 */
+	function test_wp_enqueue_script_with_html5_support_does_not_contain_type_attribute() {
+		add_theme_support( 'html5', array( 'script' ) );
+
+		$GLOBALS['wp_scripts']                  = new WP_Scripts();
+		$GLOBALS['wp_scripts']->default_version = get_bloginfo( 'version' );
```

Comment:
Not that this PR should be changing this, but shouldn't the default version be `classicpress_asset_version()`?

Reply:
It really doesn't matter in the unit tests as the version is defined twice for the test later, that said we should probably set a good example of how to use `classicpress_asset_version()`, PR update to follow.